src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -407,7 +407,107 @@ After configuring the OAuth endpoints (with Cognito Hosted UI), you can integrat
 
 A custom state is not required, but if you are looking to add one, you are able to do so by passing a string value (e.g. `signInWithRedirect({ customState: 'xyz' })`) and listening for the custom state via [Hub](/[platform]/build-a-backend/utilities/hub/). You can also use the Hub eventing system to catch errors using `signInWithRedirect()` to listen for the `signInWithRedirect_failure` event.
 
-<InlineFilter filters={["javascript"]}>
+<InlineFilter filters={["javascript", "angular", "vue"]}>
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Amplify } from "aws-amplify";
+import { Hub } from "aws-amplify/utils";
+import { signInWithRedirect, getCurrentUser, AuthUser } from "aws-amplify/auth";
+
+import amplifyconfig from "./amplifyconfiguration.json";
+
+Amplify.configure(amplifyconfig);
```

Comment:
Do we need to include this part? Won't the customer have already done this somewhere else in their app?